public/js/showoff.js

```diff
@@ -131,6 +133,8 @@ function initializePresentation(prefix) {
 	}
 	setupSlideParamsCheck();
 
+  // Remove spinner in case we're reloading
+  $('html,body').css('cursor','');
```

Comment:
You can change it if you want. There's hardly any benefit to doing either.

eh, I'll do it. You've got other things. Gimme a minute